core/domain/topic_domain.py

```diff
@@ -221,14 +236,107 @@ def validate(self):
                     % (skill_id, self.id))
 
 
+class SubtopicPage(object):
```

Comment:
It's OK to move this to a separate domain file altogether dedicated to subtopic pages. Otherwise it looks a bit like this is a substructure of Topic, when it's not.

Reply:
Done.